CoffeeChat/Controller/EditInterestViewController.swift

```diff
@@ -0,0 +1,195 @@
+        Interest(name: "Fashion", categories: "lorem, lorem, lorem, lorem, lorem", image: "fashion"),
+        Interest(name: "Fitness", categories: "lorem, lorem, lorem, lorem, lorem", image: "fitness"),
+        Interest(name: "Food", categories: "lorem, lorem, lorem, lorem, lorem", image: "food"),
+        Interest(name: "Humanities", categories: "lorem, lorem, lorem, lorem, lorem", image: "humanities"),
+        Interest(name: "Music", categories: "lorem, lorem, lorem, lorem, lorem", image: "music"),
+        Interest(name: "Photography", categories: "lorem, lorem, lorem, lorem, lorem", image: "photography"),
+        Interest(name: "Reading", categories: "lorem, lorem, lorem, lorem, lorem", image: "reading"),
+        Interest(name: "Sustainability", categories: "lorem, lorem, lorem, lorem, lorem", image: "sustainability"),
+        Interest(name: "Technology", categories: "lorem, lorem, lorem, lorem, lorem", image: "tech"),
+        Interest(name: "Travel", categories: "lorem, lorem, lorem, lorem, lorem", image: "travel"),
+        Interest(name: "TV & Film", categories: "lorem, lorem, lorem, lorem, lorem", image: "tvfilm")
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+
+        interestsTableView.delegate = self
+        interestsTableView.dataSource = self
+        interestsTableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.reuseIdentifier)
+        interestsTableView.isScrollEnabled = true
+        interestsTableView.clipsToBounds = true
+        interestsTableView.backgroundColor = .none
+        interestsTableView.allowsMultipleSelection = true
+        interestsTableView.bounces = false
+        interestsTableView.showsHorizontalScrollIndicator = false
+        interestsTableView.showsVerticalScrollIndicator = false
+        interestsTableView.separatorStyle = .none
+        interestsTableView.sectionFooterHeight = 0
+        view.addSubview(interestsTableView)
+
+        interestsTableView.snp.makeConstraints { make in
+            make.leading.trailing.equalTo(view.safeAreaLayoutGuide).inset(39)
+            make.top.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+    }
+
+    // MARK: - Data Management
+    private func getInterestFromIndexPath(_ indexPath: IndexPath) -> Interest {
+        return indexPath.section < yourInterests.count
+            ? yourInterests[indexPath.section]
+            : moreInterests[indexPath.section - yourInterests.count]
+    }
+
+}
+
+// MARK: - UITableViewDelegate
+extension EditInterestViewController: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let interest = getInterestFromIndexPath(indexPath)
+        if indexPath.section < yourInterests.count { // From Top Section
+            yourInterests.removeAll(where: { $0.name == interest.name })
```

Comment:
do we need a `removeAll`? could we use a `removeFirst`? not sure if its possible for interests to have the same name though